src/ion/liveranges.rs

```diff
@@ -227,31 +192,26 @@ impl<'a, F: Function> Env<'a, F> {
             );
         }
 
-        if self.vregs[vreg.index()].ranges.is_empty()
+        if self.vregs[vreg].ranges.is_empty()
             || range.to
-                < self.ranges[self.vregs[vreg.index()]
-                    .ranges
-                    .last()
-                    .unwrap()
-                    .index
-                    .index()]
```

Comment:
This is *so* much nicer with less stuttering! "vreg vreg index ranges last unwrap index index" reads like some wild Forth sequence or something...